beacon_node/beacon_chain/src/metrics.rs

```diff
@@ -1873,15 +1834,6 @@ pub static BLOCK_PRODUCTION_BLOBS_VERIFICATION_TIMES: LazyLock<Result<Histogram>
     )
     },
 );
-/*
- * Availability related metrics
- */
-pub static BLOCK_AVAILABILITY_DELAY: LazyLock<Result<IntGauge>> = LazyLock::new(|| {
-    try_create_int_gauge(
-        "block_availability_delay",
```

Comment:
this sounds like a useful metric that we should use

Reply:
We do have this:

https://github.com/sigp/lighthouse/blob/a4e3f361bfae83d779a1e09dd4581d01686ff33a/beacon_node/beacon_chain/src/metrics.rs#L1390-L1396

Reply:
oh nvm just read the comments below!